examples/python/mcp/mcp-http-stream.py

```
@@ -0,0 +1,104 @@
+    verbose=True
+)
+
+# Run the streaming workflow
+if __name__ == "__main__":
+    print("📡 MCP HTTP Stream Example")
+    print("This example demonstrates real HTTP streaming capabilities with MCP servers")
+    print("=" * 80)
+    
+    # Try to use real MCP HTTP streaming
+    try:
+        from praisonaiagents import MCP
```

Ruff (0.12.2)

`praisonaiagents.MCP` imported but unused

Remove unused import: `praisonaiagents.MCP`

(F401)